addons/purchase/migrations/9.0.1.2/pre-migration.py

```diff
@@ -31,7 +31,63 @@ def map_order_state(cr):
         WHERE l.order_id = o.id""")
 
 
-@openupgrade.migrate()
-def migrate(cr, version):
+def pre_create_po_double_validation(cr, env):
+    """ In v8 when you install the module 'purchase_double_validation'
+    the po workflow is altered to put conditions to change the po status to
+    waiting approval if a condition is met. In v9 this is handled without
+    workflow."""
+
+    cr.execute("""SELECT column_name
```

Comment:
I do not really need to check if the module exists. Just by checking if the workflow step added by the module 'purchase_double_validation' exists is enough.